.github/workflows/test-programs.yml

```
@@ -1,65 +1,68 @@
+              uses: metaplex-foundation/actions/cache-program@v1
+              with:
+                  folder: ./programs/${{ matrix.program }}
+                  key: program-${{ matrix.program }}
 
-      - name: Run cargo clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --all-targets --all-features --no-deps --manifest-path ./programs/${{ matrix.program }}/Cargo.toml
+            - name: Run cargo fmt
+              uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)